<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterCreationDialog</class>
 <widget class="QDialog" name="CharacterCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>514</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>741</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Character Name</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="characterNameBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Race</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Sex</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Alignment</string>
       </property>
      </widget>
      <widget class="QComboBox" name="raceComboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>201</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="sexComboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>201</width>
         <height>27</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="alignmentComboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>201</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="raceStatsButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Race Stats</string>
       </property>
      </widget>
      <widget class="QPushButton" name="guildStatsButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>260</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Guild Stats</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>241</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="strSpinBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="intSpinBox"/>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Wisdom</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Intelligence</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Charisma</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Dexterity</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="dexSpinBox"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="strengthRangeLabel">
          <property name="text">
           <string>()</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="chaSpinBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Constitution</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="wisSpinBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Strength</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="conSpinBox"/>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextBrowser" name="remainingStatPointsText">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;7&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="remainingStatPointsLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>266</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Stat Points Left</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>160</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="GuildVLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveCharacter">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>476</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Save Character</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tutorialButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>476</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Tutorial</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>476</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>characterNameBox</tabstop>
  <tabstop>raceComboBox</tabstop>
  <tabstop>sexComboBox</tabstop>
  <tabstop>alignmentComboBox</tabstop>
  <tabstop>raceStatsButton</tabstop>
  <tabstop>guildStatsButton</tabstop>
  <tabstop>strSpinBox</tabstop>
  <tabstop>intSpinBox</tabstop>
  <tabstop>wisSpinBox</tabstop>
  <tabstop>conSpinBox</tabstop>
  <tabstop>chaSpinBox</tabstop>
  <tabstop>dexSpinBox</tabstop>
  <tabstop>remainingStatPointsText</tabstop>
  <tabstop>saveCharacter</tabstop>
  <tabstop>tutorialButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
